cabal.nix

```diff
@@ -1,15 +0,0 @@
-{ mkDerivation, base, hpack, inline-r, pkgconfig, R, stdenv }:
```

Comment:
Generally we don't generate nix files directly, but instead use `callCabal2nix` as I've done in the `default.nix` file. This file is generated at build-time.  It is more convenient than having to make sure to keep it up to date.